<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xJsonViewGui</class>
 <widget class="QWidget" name="xJsonViewGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>SearchKey:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_search"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_display">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ExpandAll</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QJsonView" name="jsonView"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QJsonView</class>
   <extends>QTreeView</extends>
   <header>qjsonmodel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
